maint/tuning/coll/json_gen.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
+
+cfile="src/mpi/coll/src/csel_json_autogen.c"
+
+cat > $cfile<<EOF
+/* -*- Mode: C; c-basic-offset:4 ; -*- */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+/* DO NOT EDIT!  This file is autogenerated from the corresponding
+ * json files */
+
+EOF
+
+create_json_buf()
+{
+    json_file=$1
+    bufname=$2
+
+    echo "const char $bufname[] = {" >> $cfile
+
+    # split all the characters in the json file into a character array
+    cat ${json_file} | awk '{ printf("%s ", $0); }' | \
+        sed -e "s/[[:space:]]//g" -e "s/\(.\)/'\1',/g" >> $cfile
+    echo "0 };" >> $cfile
+    echo >> $cfile
+}
+
+# create specific json buffers
+rm -f $cfile
```

Comment:
cfile is deleted here after the 'DO NOT EDIT' header is written? Probably should be moved to before the 'cat' command.

Reply:
Yikes, you are correct.  I'll fix it.  Thanks for catching it.

Reply:
Fixed.